name,value,relevant log,explanation
io.bytes.per.checksum,1490559308,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid values: dfs.bytes-per-checksum (=1490559308) must divide block size (=134217728).
	",value-hit
dfs.datanode.ec.reconstruction.stripedread.timeout.millis,5000,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid values: dfs.bytes-per-checksum (=1490559308) must divide block size (=134217728).
	",property-item-hit	ec
yarn.client.failover-retries,0,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid values: dfs.bytes-per-checksum (=1490559308) must divide block size (=134217728).
	",value-hit
dfs.datanode.ec.reconstruction.stripedread.timeout.millis,5000,INFO	DefaultSpeculator.addSpeculativeAttempt -- we are speculating task_1701457076904_0552_m_000008,property-item-hit	ec
yarn.client.failover-retries,0,INFO	DefaultSpeculator.addSpeculativeAttempt -- we are speculating task_1701457076904_0552_m_000008,value-hit
io.bytes.per.checksum,1490559308,ERROR	Task: attempt_1701457076904_0552_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid values: dfs.bytes-per-checksum (=1490559308) must divide block size (=134217728).	,value-hit
dfs.datanode.ec.reconstruction.stripedread.timeout.millis,5000,ERROR	Task: attempt_1701457076904_0552_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid values: dfs.bytes-per-checksum (=1490559308) must divide block size (=134217728).	,property-item-hit	ec
yarn.client.failover-retries,0,ERROR	Task: attempt_1701457076904_0552_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid values: dfs.bytes-per-checksum (=1490559308) must divide block size (=134217728).	,value-hit
io.bytes.per.checksum,1490559308,INFO	Diagnostics report from attempt_1701457076904_0552_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid values: dfs.bytes-per-checksum (=1490559308) must divide block size (=134217728).	,value-hit
dfs.datanode.ec.reconstruction.stripedread.timeout.millis,5000,INFO	Diagnostics report from attempt_1701457076904_0552_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid values: dfs.bytes-per-checksum (=1490559308) must divide block size (=134217728).	,property-item-hit	ec
yarn.client.failover-retries,0,INFO	Diagnostics report from attempt_1701457076904_0552_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid values: dfs.bytes-per-checksum (=1490559308) must divide block size (=134217728).	,value-hit
io.bytes.per.checksum,1490559308,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	io
yarn.nodemanager.runtime.linux.docker.stop.grace-period,10,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	stop
io.bytes.per.checksum,1490559308,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	io
yarn.nodemanager.runtime.linux.docker.stop.grace-period,10,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	stop
io.bytes.per.checksum,1490559308,"WARN	Graceful stop failed. Exiting..java.lang.NullPointerException
	",property-item-hit	io
yarn.nodemanager.runtime.linux.docker.stop.grace-period,10,"WARN	Graceful stop failed. Exiting..java.lang.NullPointerException
	",property-item-hit	stop
io.bytes.per.checksum,1490559308,INFO	Exiting with status 1: java.lang.NullPointerException,property-item-hit	io
